com.woltlab.wcf/templates/__audioAttachmentBBCode.tpl

```diff
@@ -0,0 +1,20 @@
+<span class="mediaBBCode">
+	<audio src="{$attachment->getLink()}" style="display: none;" id="attachmentAudio_{$attachmentIdentifier}" controls></audio>
+	
+	<span class="mediaBBCodeCaption">
+		<a href="{$attachment->getLink()}">{$attachment->filename}</a>
+	</span>
+</span>
+
+<script data-relocate="true">
+	{* try to determine if browser might be able to play video *}
+	var audio = elById('attachmentAudio_{@$attachmentIdentifier}');
+	var canPlayType = elCreate('audio').canPlayType('{$attachment->fileType}');
```

Comment:
This leaks globals (probably also within the video attachment template). Wrap this in an IIFE.